### ISSUE TITLE
Stopping folder "cv2d3-pzjRv" - folder marker missing

### ISSUE THREAD

**Issue by JujuLand** [syncthing/syncthing#3309]
Hi,

I have this problem :

2016-06-15 16:15:43: Stopping folder "cv2d3-pzjRv" - folder marker missing

I have two computers, ordi1 with user1 and ordi2 with user2 and user3
user1 share folder1 with user2 and user3

When I put it in place, I first work with user1
Then I work with user2, and synchronize between folder1 and folder2 => ok (now user1 has folder1 and folder2)
I disconnect user2, and connect user3
I work with user3 and synchronize between folder1 and folder3 => it beguns, but stop at 94%, and I have this message.
The message appears about a subfolder originally in folder2 of user2

.stfolder is not missing in folder1 of user1

If you have an idea about my problem.

Thanks
A+

**Comment by calmh** [comment]
I can't follow the above, but the error is fairly clear in that syncthing can no longer see the .stfolder marker in the folder root on the mention folder. You're talking about disconnecting and stuff, if that means logging out and you are using encrypted home directories the that is usually the problem. I suggest you open a support request on the forum as this looks unlikely to be a bug.

**Comment by JujuLand** [comment]
No, .stfolder is present in the root sync folder

No encrypted directories

So, for me, I'll think it's a bug.
Perhaps due to the sync of the same folder to two users af a same computer.

I will remove the sync, and all trace and try to sync again.

A+

**Comment by AudriusButkevicius** [comment]
Does it have permissioms to read .stfolder?

**Comment by JujuLand** [comment]
Hi,
Thinking the problem was in folder1, I have a look in it, and t.stfolder was here and with good rights.

Having a look in folder3, I saw it was missing.
Adding it solves the problem.

I just notice a funny thing.
No graphic connection of user1, just an ssh connection.
Having a graphic connection (teamviewer) of user3 was enough to start syncking. Obviously after the creation of .stfolder.

I have found no difference between user2 and user3 which can explain why .stfolder was deleted.

Thanks
A+
